lib/castable.h

```diff
@@ -32,12 +32,18 @@ class ICastable {
         return to<T>() != nullptr;
     }
 
-    /// Tries to convert the class to type T. Returns a nullptr if the cast fails.
+    /// Performs a checked cast. A BUG occurs if the cast fails.
```

Comment:
`std::bad_cast` instead?

Reply:
Let's leave it as-is. As I said, `custom-rtti` branch implements it via `checkedCast`. So we won't change the comment all the time.

Reply:
Mostly asking because it might take a while longer until the other PR is merged. But the comment was wrong in the first place and this is an improvement... 

BUG and std::bad_cast are close enough in behavior so it is fine.